### PR TITLE
lvs: add Chinese translation

### PR DESCRIPTION
**Pull request by NealSCarffery**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by NealSCarffery** [comment]
Add a tldr for lvm command for Chinese User.

File: pages.zh/linux/lvm.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# lvs
```
Comment (navarroaxel) [inline pages.zh/linux/lvm.md:1]: The filename is `lvm` but the page is for `lvs`.

File: pages.zh/linux/lvm.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+> 显示逻辑卷信息。
+> 另见: `lvm`。
+> 更多信息参考:<https://man7.org/linux/man-pages/man8/lvs.8.html>。
+
+- 显示逻辑卷信息
+`lvs`
+
+- 显示所有逻辑卷：
+`lvs -a`
```
Comment (navarroaxel) [inline pages.zh/linux/lvm.md:11]: You need to add an empty line between the example description and the example itself.
```suggestion
- 显示所有逻辑卷：

`lvs -a`
```

**Review by NealSCarffery — COMMENTED** [review]
already fix title and add empty lines

File: pages.zh/linux/lvm.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# lvs
+
+> 显示逻辑卷信息。
+> 另见: `lvm`。
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:4]:
```suggestion
> 另见: `lvm`.
```

See rule 5 in [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.zh.md#%E4%B8%AD%E8%A5%BF%E6%96%87%E6%B7%B7%E6%8E%92%E8%A7%84%E5%88%99).

File: pages.zh/linux/lvm.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+> 更多信息参考:<https://man7.org/linux/man-pages/man8/lvs.8.html>。
+
+- 显示逻辑卷信息
+`lvs`
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:8]:
```suggestion
- 显示逻辑卷信息：

`lvs`
```

File: pages.zh/linux/lvm.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- 显示所有逻辑卷：
+
+`lvs -a`
+
+- 修改默认显示，以展示更多信息:
+`lvs -v`
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:15]:
```suggestion
- 修改默认显示，以展示更多信息：

`lvs -v`
```

File: pages.zh/linux/lvm.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# lvs
+
+> 显示逻辑卷信息。
+> 另见: `lvm`。
+> 更多信息参考:<https://man7.org/linux/man-pages/man8/lvs.8.html>。
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:5]:
```suggestion
> 更多信息参考:<https://man7.org/linux/man-pages/man8/lvs.8.html>.
```

File: pages.zh/linux/lvm.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+- 显示所有逻辑卷：
+
+`lvs -a`
+
+- 修改默认显示，以展示更多信息:
+`lvs -v`
+
+- 只显示特定信息域:
+`lvs -o {{field_name_1}},{{field_name_2}}`
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:18]:
```suggestion
- 只显示特定信息域：

`lvs -o {{field_name_1}},{{field_name_2}}`
```

File: pages.zh/linux/lvm.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+- 附加域到默认展示：
+`lvs -o +{{field_name}}`
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:21]:
```suggestion
- 附加域到默认展示：

`lvs -o +{{域名}}`
```

File: pages.zh/linux/lvm.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+- 取缔标题线:
+`lvs --noheadings`
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:24]:
```suggestion
- 抑制标题行：

`lvs --noheadings`
```

File: pages.zh/linux/lvm.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+- 使用指定分隔符分隔域:
+`lvs --separator {{=}}`
```
Comment (blueskyson) [inline pages.zh/linux/lvm.md:27]:
```suggestion
- 使用指定分隔符分隔域：

`lvs --separator {{=}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for your contribution. We have a Chinese [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.zh.md) for translation.

**Comment by NealSCarffery** [comment]
I want try again.

**Pull request closed without merging**